utils/common.js

```diff
@@ -0,0 +1,13 @@
+function hasFacebookCredentials () {
+    return process.env.FB_APP_ID !== undefined &&
+        process.env.FB_APP_SECRET !== undefined &&
+        process.env.FB_CALLBACK_URL !== undefined;
```

Comment:
Should we check for something "falsy" instead? Leaving just the `FB_APP_ID=` definition (as it currently is in `.env.local`) sets these env variables to the empty string.